src/plugins/abrt-retrace-client.c

```diff
@@ -119,8 +119,9 @@ static int create_archive(bool unlink_temp)
     int tempfd = mkstemps(filename, /*suffixlen:*/7);
     if (tempfd == -1)
         perror_msg_and_die(_("Can't create temporary file in "LARGE_DATA_TMP_DIR));
-    if (unlink_temp)
-        g_unlink(filename);
+    if (unlink_temp && g_unlink(filename) != 0) {
```

Comment:
FWIW, this file seems to be using a newline before the opening brace after an `if`.

Reply:
Good catch. It's a mine field out there :roll_eyes: